docs/NSR.md

```diff
@@ -457,37 +457,79 @@ The CA SHOULD ensure incident response plans minimally include:
    2. containment of the incident to minimize further impact; and
    3. identification and mitigation or eradication of the incident root cause(s).
 
-### 4. Vulnerability Detection and Patch Management
+### 4. Vulnerability Management
 
-Certification Authorities and Delegated Third Parties SHALL:
+The CA MUST implement the policies and procedures in this Section for identifying, evaluating, and resolving security vulnerabilities.
 
-a. Implement intrusion detection and prevention controls under the control of CA or Delegated Third Party Trusted Roles to protect Certificate Systems against common network and system threats;
+These policies and procedures MUST apply to all Certificate Systems.
 
-b. Document and follow a vulnerability correction process that addresses the identification, review, response, and remediation of vulnerabilities;
+These policies and procedures SHOULD apply to Security Support Systems.
 
-c. Undergo or perform a Vulnerability Scan
+#### 4.1 Inventory of Certificate Systems
 
-   1. within one (1) week of receiving a request from the CA/Browser Forum,
-   2. after any system or network changes that the CA determines are significant, and
-   3. at least every three (3) months, on public and private IP addresses identified by the CA or Delegated Third Party as the CA’s or Delegated Third Party’s Certificate Systems;
+The CA MUST define an inventory of Certificate Systems.
 
-d. Undergo a Penetration Test on the CA’s and each Delegated Third Party’s Certificate Systems on at least an annual basis and after infrastructure or application upgrades or modifications that the CA determines are significant;
+#### 4.2 Intrusion Detection and Prevention
 
-e. Record evidence that each Vulnerability Scan and Penetration Test was performed by a person or entity (or collective group thereof) with the skills, tools, proficiency, code of ethics, and independence necessary to provide a reliable Vulnerability Scan or Penetration Test; and
+The CA MUST protect the systems in the inventory of Certificate Systems against common network and system threats using intrusion detection and prevention controls.
 
-f. Do one of the following within ninety-six (96) hours of discovery of a Critical Vulnerability not previously addressed by the CA’s vulnerability correction process:
+Some common network and system threats include, but are not limited to:
 
-   1. Remediate the Critical Vulnerability;
-   2. If remediation of the Critical Vulnerability within ninety-six (96) hours is not possible, create and implement a plan to mitigate the Critical Vulnerability, giving priority to
+* malicious software;
+* phishing and social engineering;
+* denial of service;
+* unauthorized access; and
+* malicious data injection.
```

Comment:
This would leave 4.2 as just
> The CA MUST protect the systems in the inventory of Certificate Systems against common network and system threats using intrusion detection and prevention controls.

Do you think a lack of guidance or explanation regarding what "common network and system threats" entails will cause confusion and/or inconsistency in meeting the requirements? Similarly, do you think the current list introduces confusion or causes other issues?

Can you provide any more detail about what points of discussion led to favoring removal of the list? To be clear, I'm not particularly opposed to removing it, but I don't have anything better to put in its place, so understanding why your discussion concluded as it did would be great.

Reply:
Yes, that brief version of 4.2 is exactly what is suggested. Our discussion centered around the desire to keep the things more general and not get into little pockets of being unnecessarily specific. This felt oddly specific to us and didn't clear up anything that isn't super basic for anyone doing threat identification. It's not harmful, but it didn't really feel helpful either. So, it felt like it should be cut.

Reply:
List removed based on consensus in NSWG call 10 Sep 2024